IOSExercises/Classes/EXRPerson.h

```diff
@@ -0,0 +1,33 @@
+//
+//  EXRPerson.h
+//  IOSExercises
+//
+//  Created by Arnold Joseph Caesar Esteban on 11/26/20.
+//  Copyright © 2020 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "EXRColor.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface EXRPerson : NSObject
+
+@property (strong, readwrite, nonatomic) NSString *image;
```

Comment:
Best to rename since an `image` string is not so intuitive (imageFilename? imageFilepath?)